js/client-topbar.js

```diff
@@ -612,7 +612,7 @@
 			var ctrlPlus = '<kbd>' + (navigator.platform === 'MacIntel' ? 'Cmd' : 'Ctrl') + '</kbd> + ';
 			buf += '<p class="optlabel">**<strong>bold</strong>** (' + ctrlPlus + '<kbd>B</kbd>)</p>';
 			buf += '<p class="optlabel">__<em>italics</em>__ (' + ctrlPlus + '<bkd>I</kbd>)</p>';
-			buf += '<p class="optlabel">``<code>code formatting</code>``</p>';
+			buf += '<p class="optlabel">``<code>code formatting</code>`` (' + ctrlPlus + '<bkd>`</kbd>)</p>';
```

Comment:
```suggestion
			buf += '<p class="optlabel">``<code>code formatting</code>`` (<kbd>Ctrl</kbd> + <kbd>`</kbd>)</p>';
```

You spelled `kbd` wrong, but also Cmd+` is specifically "switch window" on Mac and won't work.

Reply:
wtf. i literally just did a copy and paste? how did it get transposed? thanks.